simple_use_case/source/poisson.py

```diff
@@ -88,7 +88,8 @@ def solve_and_write_output(mesh: str, degree: int, outputfile: str, numdofs: str
     PARSER.add_argument(
         "-n",
         "--num-dofs",
-        required=True,
+        required=False,
+        default="dummy_num_dofs.txt",
```

Comment:
I wonder why this appears as diff, I thought I had already added this to #32 ?

Reply:
I also wondered about that. This branch was created before the change was made in #32 .
I then rebased this branch to #32.

Reply:
somehow the commit was duplicated? this is the original one https://github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/pull/32/commits/c9c76c45f37009f90c43db237666153bed74fbe9 and this is the one in this PR https://github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/pull/37/commits/d8ff82ad37e9eccf34b1458e02a7751c6d0ce238

Reply:
I rebased again and force pushed the changes. Do you agree to merge?

Reply:
looks good, I'll merge